R/plots.R

```diff
@@ -474,5 +572,28 @@ faceted_time_series_plot <- function(sensor_concentrations, wind_data, time_sequ
 #' print(plot)

```

Comment:
In R, no need to explicitly `print()` a plot as you might in Python. Rather, just return the object by making a direct call to it, e.g., `plot`